testing/acceptance_test.py

```diff
@@ -646,6 +646,12 @@ def test_pyargs_only_imported_once(self, testdir):
         # should only configure once
         assert result.outlines.count("configuring") == 1
 
+    def test_pyargs_filename_looks_like_module(self, testdir):
+        testdir.tmpdir.join("conftest.py").ensure()
+        testdir.tmpdir.join("t.py").write("def test(): pass")
+        result = testdir.runpytest("--pyargs", "t.py")
```

Comment:
Hmm what happens here? Does it run the file? Is that what's expected?

Reply:
it runs the file, yes -- it's at least consistent with the 4.6.x behaviour -- I didn't really consider whether this is sane / correct or not :laughing: 

I've never really worked with `--pyargs` so I'm not sure what the expected thing is here

Reply:
Well if it works like 4.6.x then that's fine by me! 

Thanks for the quick fix!